SimCalorimetry/HGCalSimProducers/python/hgcalDigitizer_cfi.py

```diff
@@ -233,17 +283,15 @@ def HGCal_setRealisticNoiseSi(process,byDose=True,byDoseAlgo=0):
         )
     return process
 
-def HGCal_setRealisticNoiseSci(process,byDose=True,byDoseAlgo=0):
+def HGCal_setRealisticNoiseSci(process,byDose=True,byDoseAlgo=0,byDoseMap=cms.string("SimCalorimetry/HGCalSimProducers/data/doseParams_3000fb_fluka-3.7.20.txt")):
```

Comment:
I'm not sure spreading the `byDoseMap` as default parameter in so many different places is desirable.
Wouldn't it better to define it once and reference it everywhere else?

Reply:
ok